raincloud/models/service.py

```diff
@@ -33,27 +34,31 @@ def set_status(self, status):
         self.status = self.get_status()
 
     def get_status(self):
-        # try to attach to logs
         client = docker.from_env()
-
         try:
             if client.containers.get(self.name).status == 'running':
                 return 'enabled'
             else:
                 return 'disabled'
         except docker.errors.NotFound:
             return 'disabled'
-    
+
+    def get_settings(self):
+        settingsFile = "{}/{}/{}".format(self.__services_folder(), self.name, "service.json")
+        if os.path.isfile(settingsFile):
+            return open(settingsFile, "r").read()
+        return {}
```

Comment:
We should go ahead and return JSON here,

```suggestion
    service_file = "{0}/service.json".format(self.__service_folder())
        if os.path.isfile(settingsFile):
            with open(service_file) as f:
                return json.load(f)
        return {}
```